pkg/parser/cse_cmd.sh.gtpl

```diff
@@ -1,3 +1,10 @@
-#!/bin/bash
-echo "hello"
-#This is a simple placeholder script, but will eventually resemble what cse_cmd.sh looks today
\ No newline at end of file
+ADMINUSER = "{{.LinuxAdminUsername}}"
+PROVISION_OUTPUT = "{{.ProvisionOutput}}"
+REPO_DEPOT_ENDPOINT ="{{.RepoDepotEndpoint}}"
```

Comment:
lets try to imitate the actual cse_cmd.sh as much as possible, the repo depot endpoint had a conditional on custom cloud, we should include that too

Reply:
@anujmaheshwari1 I added a {{if}} to the REPO_DEPOT_ENDPOINT. 
And I add a comment saying that {{/* need to properly add {{GetInitAKSCustomCloudFilepath}} >> /var/log/azure/cluster-provision.log 2>&1; */}}